### DISCUSSION TITLE
Win 11 - Frequently crashing

### DISCUSSION THREAD

**Discussion by merexodram** [sumatrapdfreader/sumatrapdf#3098] [Q&A]
Hello All,
I've been using Sumatra for years now, linked with eclipse and pdflatex. Recently, although it is quite in the beginning of my paper -meaning it is very light in content yet- it became frequently crashing like at least one of a three compiles. I'm sure win 11 is the main reason, but I still wanted to ask whether anybody else came up with the same thing.
Cheers,
Mert

**Comment by GitHubRulesOK** [comment]
There are many reasons for instability in such a configuration so it is hard to say.
Java eclipse ? has many components that can be affected
PdfLaTex is built on several Distros with different update cycles and dependencies

SumatraPDF is best added as one consistent exe to reduce its permutations, 
and the most stable for historic LaTeX (thus reliable) version is undoubtedly 32bit 3.1.2 portable from SematraPDF.org,
however even that had a locking limitation around 10MB of pdf
Recent Pre-Release or portable (64bit installed has open related issue) has higher locking limitation around 32 mb

So check version and is it installed or portable ? is it 32bit or 64bit are PDFs near to 30MB

**Reply by merexodram** [reply]
Hello,

Actually my pdf's are quite small (that's why I wanted to mention they are "light in content") around 400KB at the moment. I use Texclipse to compile PdfLatex and automatic configuration runs SumatraPDF at each save attempt. 

I was always using 64bit installed version, thought that would be best in any case. But, as you suggested, I tried portable 32bit and up to now it seems quite good. I'll post any update.

Thank you for your kind and quick response,

**Reply by GitHubRulesOK** [reply]
the issue with 64 bit install is down to a dependency on compression libraries to support the synctex file that allows indexed positions between editor tex lines and pdf work areas, so it is troublesome when compression is on for some users. Stable with uncompressed syctex files or perhaps ok when 64bit is not installed, but stable with 32 bits as portable !!
